rpm/generic/zfs.spec.in

```diff
@@ -1,5 +1,13 @@
 %global _sbindir    /sbin
-%global _libdir     /%{_lib}
+
+# Set the default libdir directory based on distribution.
+%if %{undefined _libdir}
+%if 0%{?fedora} >= 17 || 0%{?rhel} >= 7 || 0%{?centos} >= 7
+%global _libdir    /%{_lib}
+%else
+%global _libdir    /lib
+%endif
```

Comment:
I believe you're going to have to add `--with-libdir=%{_libdir}` to the `%configure` macro later on in the file.

Reply:
Actually, never mind. The `%configure` macro will pick it up automatically. My mistake.

Reply:
Yes, exactly, it's automatically added, so no need to add it